.github/actions/build-autoreduction/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Build Autoreduction in a test enviroment'
+description: 'Intial steps to build Autoreduction in a test enviroment'
+runs:
+  using: "composite"
+  steps:       
+    - name: Verify MySQL connection from host
+      run: |
+        sudo apt-get install -y mysql-client
+        mysql --host 127.0.0.1 --port 3306 -u root -e "SHOW DATABASES"
+      shell: bash
+
+    - name: Set up database
+      run: mysql --host 127.0.0.1 --port 3306 -u root < build/database/reset_autoreduction_db.sql
+      shell: bash
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
I don't think you can have this in the Action - that's what it's complaining about. But you should be able to stick this in the workflow before the action call